cloudinit/util.py

```diff
@@ -2782,7 +2782,14 @@ def get_mount_info(path, log=LOG, get_mnt_opts=False):
 
 
 def has_mount_opt(path, opt: str) -> bool:
-    *_, mnt_opts = get_mount_info(path, get_mnt_opts=True)
+    try:
+        *_, mnt_opts = get_mount_info(path, get_mnt_opts=True)
```

Comment:
why would this function return `None` to begin with?

Reply:
> why would this function return `None` to begin with?

The reason still needs to be further identified. However, when we look at the implementation code of this function, there is some cases of returning None.

Reply:
of all of those, only the first and the last would leave no trace in the log

the first one would mean your mountinfo file is in the wrong, or a new unknown, different, format.

the last one would mean no match was found, including no match for `/`!
I'm not sure there are scenarios where that makes sense (at least as a system to run cloud-init on)